meta-lmp-base/recipes-bsp/u-boot/u-boot-tools_2022.01.bbappend

```diff
@@ -1,6 +1,5 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
 
 # Remove CVE-2021-27138 fixes when using '@' as fit separator
-SRC_URI:remove = '${@oe.utils.conditional("FIT_NODE_SEPARATOR", "@", "\
-	file://CVE-2021-27138-1.patch \
+SRC_URI:remove = 'file://CVE-2021-27138-1.patch \
```

Comment:
Since we are removing support for '@' as separators, we should remove this SRC_URI line.